tools/run_tests/python_utils/port_server.py

```diff
@@ -33,18 +33,20 @@
 from __future__ import print_function
 
 import argparse
-from six.moves import BaseHTTPServer
+from BaseHTTPServer import HTTPServer, BaseHTTPRequestHandler
```

Comment:
If I implemented this server in Go, would you be asking for me to write preprocessor macros so I could also compile it in Fortran?

We get to choose the language we use for our infrastructure, and I see no point in trying to make each program run under more than one language. Doing so just adds complexity for no real value.